stdlib/2/md5.pyi

```diff
@@ -1,10 +1,6 @@
 # Stubs for Python 2.7 md5 stdlib module
 
-class md5(object):
-    def update(self, arg: str) -> None: ...
-    def digest(self) -> str: ...
-    def hexdigest(self) -> str: ...
-    def copy(self) -> md5: ...
+from hashlib import _hash as md5
```

Comment:
Should be `from hashlib import md5 as md5`, it's an instance of `_hash` not the class itself.